Makefile

```diff
@@ -71,15 +73,6 @@ fallthrough: submodules
 	@echo Initial setup complete. Running make again . . .
 	@make
 
-go.test.unit: $(KUBEBUILDER)
```

Comment:
`go.test.unit: $(KUBEBUILDER)` is no longer required, since all kubebuilder unittests are removed